README.md

```diff
@@ -17,22 +17,24 @@ Content Roles
 Provisioning:
 
 * **user/profile** store facts to .profile and .bashrc 
-* **postgresql** install a PostgreSQL 9.3 database
+* [**postgresql**](postgresql) install a PostgreSQL 9.3 database
```

Comment:
I like these links. For GitHub they help a lot. But I fear that Ansible Galaxy will not like them that much and create links that lead no where.
a) we add full links to github
b) not use links